internal/receiver/scriptreceiver/README.md

```diff
@@ -0,0 +1,84 @@
+| sshdChecker.sh	     | Parses sshd_config for information local sshd configurations                                                                                                                                                                 |
+| time.sh	             | System date and time, and NTP server time via the shell commands and chronyc, date andntpdate                                                                                                                                |
+| top.sh	             | List of running system processes via the shell commands ps and top                                                                                                                                                           |
+| update.sh             | 	Available software updates for installed packages via the shell commands softwareupdate and yum                                                                                                                             |
+| uptime.sh             | 	System date and uptime information via the shell command date                                                                                                                                                               |
+| usersWithLoginPrivs.sh| 	Shows system username information                                                                                                                                                                                           |
+| version.sh	         | OS version details via the shell command uname                                                                                                                                                                               |
+| vmstat.sh	         | Process-related memory usage information via the shell commands prstat, prtconf, ps, sar, svmon, swap, swapinfo, sysctl, top, uptime, and vmstat                                                                             |
+| vsftpdChecker.sh      | 	Parses vsftpd.conf for information about local VSFTP server configurations in /etc, /etc/vsftpd, or /private/etc                                                                                                            |
+| who.sh	             | Information about all users currently logged in via the shell command who                                                                                                                                                    |
+
+
+## Configuration
+
+The following settings are required:
+
+- `exec_file` : Name of the script to be executed. Available script: bandwidth.sh, cpu.sh, df.sh, hardware.sh, interfaces.sh, iostat.sh, lastlog.sh, lsof.sh, netstat.sh, nfsiostat.sh, openPorts.sh, openPortsEnhanced.sh, package.sh, passwd.sh, protocol.sh, ps.sh, rlog.sh, selinuxChecker.sh, service.sh, setup.sh, sshdChecker.sh, time.sh, top.sh, update.sh, uptime.sh, usersWithLoginPrivs.sh, version.sh, vmstat.sh, vsftpdChecker.sh, who.sh
```

Comment:
Since we won't want arbitrary file execution, `script_name` or similar would be my suggestion